docs/openapi/components/schemas/common/Template.yml

```diff
@@ -0,0 +1,227 @@
+$linkedData:
+  term: Template
+  '@id': https://w3id.org/traceability#Template
+title: Template
+description: A template for rendering schemas for example with JSON Schema Forms.
+type: object
+properties:
+  type:
+    oneOf:
+      - type: array
+        items:
+          type: string
+          enum:
+            - Template
+      - type: string
+        const:
+          - Template 
+  id:
+    type: string
+  industryTags:
+    title: Industry Tags
+    description: Industry vertical applicability.
+    type: array
+    items:
+      type: string
+  industryRoleTags:
+    title: Industry Role Tags
+    description: Industry role applicability.
+    type: array
+    items:
+      type: string
+  tags:
+    title: Tags
+    description: Tags applicable to the template.
+    type: array
+    items:
+      type: string
+  schema:
+    title: Schema
+    description: The JSON Schema.
+    type: object
+  uiSchema: 
+    title: UI Schema
+    description: Layout definitions
+    type: object
+  formData:
+    title: Form Data
+    description: Placeholder example data.
+    type: object
+  organization:
+    title: Organization
+    description: The organization creating the template.
+    $ref: ./Organization.yml
+  created:
+    title: Created
+    description: Template creation date.
+    type: string
+  updated:
+    title: Updated
+    description: Template update date.
+    type: string
+required:
+  - id
+  - industryTags
+  - industryRoleTags
+  - tags
+  - schema
+  - uiSchema
```

Comment:
I think uiSchema shouldn't be required

Reply:
agree... also wondering if we should derive it from the $linkedData approach or include it in parallel like this.